gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcTransportChannel.java

```diff
@@ -91,6 +92,11 @@ public boolean awaitTermination(long duration, TimeUnit unit) throws Interrupted
   @Override
   public void close() {
     getManagedChannel().shutdown();
+    try {
+      getManagedChannel().awaitTermination(6, TimeUnit.MINUTES);
```

Comment:
Where did the 6 minutes number come from?

Reply:
If you think that is reasonable, that's fine. This is a non-blocking call then I assume?

Reply:
This is mostly long living connection. Ideally this should happen at the time when application is shutting down either user manually calls close or jre does as part of auto closeable resource. 

We have all our rpc time outs less than 1 min(mostly 10-15 sec) and all our samples/IT tests uses 1 min timeout. so anything more than 1 min should be reasonable. This is blocking call but mostly will be at time of shutting down application or resource goes out of scope when used in try with resource.